lib/brady.ex

```diff
@@ -20,6 +20,48 @@ defmodule Brady do
     ""
   end
 
+  @doc """
+  Embeds an html safe raw SVG in the markup. Also takes an optional list of CSS
+  attributes and applies those to the SVG.
+
+  Ex:
+      Brady.inline_svg("test", class: "foo", "data-role": "bar") =>
+      {:safe,
+       "<svg class=\"foo\" data-role=\"bar\" height=\"100\" width=\"100\"><desc>This is a test svg</desc><circle cx=\"50\" cy=\"50\" r=\"40\" stroke=\"black\" stroke-width=\"3\" fill=\"red\"></circle></svg>"}
+  """
+  @spec inline_svg(String.t, keyword) :: String.t
+  def inline_svg(file_name, options \\ []) do
+    path = static_path(file_name)
+    case File.read(path) do
+      {:ok, file} -> render_with_options(file, options)
+      {:error, _} -> raise "No SVG found at #{path}"
+    end
+  end
+
+  defp render_with_options(markup, []), do: {:safe, markup}
+  defp render_with_options(markup, options) do
+    markup
+    |> Floki.parse
+    |> Floki.find("svg")
+    |> add_attributes(options)
+    |> Floki.raw_html
+    |> render_with_options([])
+  end
+
+  defp add_attributes([{tag_name, existing_attributes, contents}], attributes) do
+    attributes = Enum.map(attributes, fn{key, value} -> {to_string(key), value} end)
+    {tag_name, attributes ++ existing_attributes, contents}
+  end
+
+  defp static_path(file_name) do
+    path = Application.get_env(:brady, :svg_path)
```

Comment:
What do you think about:

```elixir

path = Application.get_env(:brady, :svg_path) || "web/static/svg"
```

I think that allows removing the conditional below.

Also we should check out the `Path` module to see if there is a better way to combine the paths. I think that module could prevent us from running into issues if someone used `/my_directory/` with an ending slash and then we end up with `/my_directory//file_name.svg`